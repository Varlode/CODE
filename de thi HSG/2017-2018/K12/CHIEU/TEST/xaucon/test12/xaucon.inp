iazhtpzhtoacbskcvhhdhvrgswqflphdksdehlakjmeycbrrenouekfkkinzpmjhlobpbhcqoxctxeqtawigqjvwgcickabwcqhocyhkmreppnrquvyyibvejcxncvvfjvyorywtflicylcevwjznmztkgzutcebjbhvdhhekrbszyzmfydjbofzuljfchmssmwwhfxwqqokdqglzuxejfacpqozzhvnhlgcxvgkmndzgkpfpewqltplsiaqjqq
lxkjnzqgigofmeoooewcdanqhmyfupryrxvzqadlqftbrujzcnjraqyrcfonmjabuafyaempndhoshautlkbmlindrqzkrnhcspfpywckvhbtdwqggqvqxlfuodhdkoxslkwzahcotmdfjxtzkfjcmowvjohiylsmgmajhjyoyieqvfadcwrrnvyvzumfoifjkxqcdojlrrmfgjdrfwlfdnbufdnkcpxzryyiojrncfgmihnwoouiypthnnahsm
wpfiftujfqyotukxnwkrplcefsliktjivbuxmwusijlpjvqmmqzjvpriclshzlxagtkcljinwinridunumkwstutehddxsolxvhtvtoydodhemggdxtimcfrfaqyqutczwepdbilenxfnyquuvvmxweylblgowvcnbhcmrunxflhscqsdzawsxacdwnjlfzjeigofzipbrqljgkttcerympcmhyvhflrbjppmjvmrcpegmgpnethljszzqaguww
brbpzxiavbyrhepwtilbmibezouvescnlsgovsrmisazyjkpeutfvqnbzbiivegncvcsivuecqiugtmgqzcupszzlabtpeoanbsnebjviwaipuwkqrgnnrqoivvxfkfhiewfrqkrixoddhjvlzekaystpiwcvsudomsivwlfkszbpvwatplcjgkufymttbinlarrthopybpogcwifbfssyeynqtltoesdaypmocrjzldrcclzacaoomwetgdiwc
apskehbhehgaewwcshzteghqrxmjtafdtvitcpqfnmlyvfyftimklsbypzgizldfpecxcoqlzjqdpxpxplwnmooouiayamlxrtbsvatbgddfzmeqeuwcovhzqqfzxdqgjqmewhndomikngzoilcdxduomkdvhqfxvicazdohqufubokrzpzapxlqibgdsugcxeznucwbnywgugewoatplkzobeckictcvywfkokfjojeqturxjeqirxfewbwxiy
sqemaiowqeebhuopeshputslvtfvxhnahyoucajhytrqsbaodosmrvtnhwhrattlnbbosfpnqiqprkkavvnxvqtcszuvxxammxrsqknyyokwdnzdvqsmlplxfrfgisjucuvmnanduecllyziverrubcfcjemoymnllntducwikalxbzzscvoferueeikqvwnyfgmglnilivohbjovjspydrhayqfrnavimzmamumuxowckzcedbjnteeybskipt
wkamrtohyuathniwubfwwtnenhcrtderoyepzdyphysukbckjkykkyzfdyflgxdsxyhydduadolmsjeogcicqhwydvdvaemgehhuuzdxyapkmcmjnyzadimiqhcxyznzqtljetvvsqgyuxxuwtoiexgdlptgcwosvowcdpkhqesidtuzaonbfwvapzeojxutbouoxyohadjtyhhayqwbyqlgudmmjorowsnkcytlbhmaqovjzwptjwnrfxhvlki
pvmmgytehnvibqtmvipyspitczujgjmqlzycffefxmduxyxgziuqqqfsryioqzsonhggxabyniufiqmxpyiqiisexvmqvjdqjrdfkfswuluxbkfqlugdwtkbnybjibeuxzurezomwdjonvotbeeuioozivzqmnxtqdxpupgbgwwuwfydnvijiarxpvovotvfgqupzucxgxigqbjvgyuyznmgzpkpmpcdgvtffyqadyncrgozqiqfdcfuegiekni
hjgakgraovhpqgomjnhfiopvpudfjdlsdzasrkfjapnhxbkffejgkijsafszkmivaryptnsgxtfnmmxjfpnysqqwxsgdfkjzcrrpkpogtvjklbfupixvehlkauxjoqjieordpwpnekopdquczjgdbxdfoynbrtftqaxivmwprxlccmzbabfvbhghhbplwspohfitssybfimhbokafmoiobsqnxkzwepaylbwlpajtycsqsonsvsmbmxevymtkfl
lqtipithuxvmgywaopulgprqijojrjqpvqoyiosoveknqwvsauuvnuodvjhiinixatpqdnbdlrdjbmzrgkleutqohapohqojtsbdxjqvgtsuiyprdrejbewrutyifviyrnwhkaynbtriyclikuunmqremzftpyqcqxbwyguubzrtjtahdwutcnipalcunjsotaqvxhfdlogcuzgfxophtdodligorqgbwkmcrjwarvwzyjgbttsmfrlihakkoyv
vfwldgfowhdttlxkdiubzzaissozgcbkkppfesycuyyrwilmdxihksagcbglhxjfimxjodiuleoqwsojmpuxclnxibqxwkhjeaudwmjwqkonxqbpgjausdpabikmxxrpyckjlfptbokjyaltdadpffxtabwwlwjvkninvusqzvnexmzqiyzzcxsltmobeqijrdweqnhjxsgprleybtewxgkpzclpegjxgrohpldcnmlfsjaoxremcahtcdsbdbs
kcrrpdpwpvaalhrujvdtqihdviumpufucaqpfvyvmkuxvkkzwbnlsbkcdsoymwiguycdyjunfoqzobvwpkzgualpjvdrgsveomqhfsuusxydyaqpwbdwbemkpvjgwfqzqrkpqgmozsyapstqmnscmnjmloehqzrpvgjvlmuoxzardtxpjezjxafgmlwnwcjiahparexddvdeoiurngfwmkhdoaxbnzzplmywzaucggmwxnooioeecusohpcrskl
lrbjbcvilycdbcyeoszwyapyfiftktumnibepgmmitsszyubmmlqwdcwumacncfgwppymkupoihxccgphfqtcxbtvtuabbrjanxiyghnvsfetnhkgwworhcbolmxgzqapllnpdvimmuuhkjkpcnbgndakzoohjqhqccfipgjqpgxsyfwqmubkdpzuuzbtsdmwtqgvbfspoiisnpcywdffjrsvuvizcrhgopjzxsrbakluwtexqtwqlkfeltgfko
dgnnnppsggfnqmrlzqumjmypmwrhpqwnetipnlwhlpdybdqppkiycxqjrtfutcyukekvutzzlhwjneqptxpwodjtmbfpzpxgimnbkzgttiugcammgpkjhbfgthetimjnmctizzfdqboeiwjzyrmeadephxsbixbzwiitrflyugefedcywmpemvtkaiahkgxyiqplolufwjwjxkduxznptkawojeccwlnfaqbahpkrsdqqhmoaxwexrahhkjbvqo
cdckahugjstzuvlsdocivdxqzysuzqsebtmerumchpvtvqnksmxnrmofipufgldayzivrsdmwvplqtzavmieldvywrrijwmwdzvesitrdvdkgialjsyeiypjxnnlodjdjpkniavodncdlenpfbnbjqeyzcekibyjkcdsdiuwpbcazjoeumvajikbfsdveucspbuipnjrubhjymeymcedcqmdsfhdpxetlljppklhmrkuxaaevukpkhyysxkpzjp
eeripoxencvxvoclklrkfrtoqjmzcvodrrotywsuvbwdcwsktymbazadcrjdlhhcmhgqhccomvrudunjhhsthyaotsgvnyttypozrethgpvhvnihzdufojekczdayvrmsjtxzkoijqobiwziocmppxqhceoazzhuwwoywodoypptmlgufusaqqxxdhwsazaxelwratbdadfhbbssjrhlknzvameuzpwceohftqmhinarslvqssmiicpmtpufwfq
wlxpsfbkmnvryhxczzdgumwworiyfxvbtilydlkoggjxnvkjhirahfswmygwzbvxoskujkaomlfgfowjruaylgrmziweifreblzooqouesjsntrnewyyexgxyuikxerdleklxytyblyajxoxslenolovzyqyzzqwmzhubrhiwgzpoqtfurxscxaarvzzxmkgylieqyaptxvattvtuantdkqyocnbwgkrqofxcqonduszjodrlosywabkdwxujhn
rubyowbmdzqounffdvwyhjqhllvdmjpekgffxasbuxykhesfdnfrprjlxhcsxfcrcfgkedutuokaymhbfzmuncsaybtkhxxmmgcvibjwjcrcfnxnyxdoftliovxojjhyyoqplelxegpifoxkusjpfubnezvgkzgahcrnsgecinbidnaryrsuspxtciabgqwezewhipwcxlcvkdjscrkrjmmwrshgtntixmzcoeogwjfaqxmeiotzddqekfyplas
terboeeqmocxpfvpdgfqmiewcxxwwnzmajobyltcgbjqvajtnppzrcdmhxmnwxjwstvqitrehutyopfkpihhcfakcizugwnekamhephegauwetptlylqcshyfzregoaudoitxiubvvsdkcqatdogqxqoepzmsxdwwpkdalcznzvqyqqzxydmphrrkkfvvpvpajcemdxxdptxlbsgqlygctwtfykxeneaxgcbbxukytegqsbzxilslruauzntusa
trtpyzckcbuzrcyznkuzsnngpvsqkhqxcjwjpprqbwacfijfhygfehajvdcyoaknftaikmiiqhokqzpmbrygwvjzuaugfopjdtpgjndomptyiwxupksvpfccjrtzihbiisrayywnhalgukqlhkvrjdaebsqwkrcostasxhxrsdnjibaouedcqsvzxzbxroqcomiyxnthjcdrkqxejsmtoiuujvkgmipwfhdbgbopaptyloshmtcksahzcgjxrwg
apyuvgzhlmdmywqbwiejyzoljscwqozuvyhnljbwgqirtiljcgbketgkkpcxmxoypglzuithrxcxbflafxkecntnvsiiitopjwfvtvwuigijncywhznitwjrgkxdvsiyylprtabpkcqnxswrwlwkvdivrfrahzetyfrwzrfpvpynsrannwqnqgkpvltzazyktrrdmkswxqnjszdqvtblikjgpglakmhfbsfmexszdazjlowsblwctjgwcpvkjco
dcxbuudpoidraadsqurydypfamnxyoforlzgjtcmxqfeujtzpeqdeygaoycbhxgvflmakkwvieqephkbgblthdgjlpmvuyrsctqzgbervsllkpfgbaazpajqcpvzrarsztgxffhbnxvvgieaamswomnpcfmuetcgpswrsebrlkbomuptrvivgzvybdukccjzkrwoaedzsqzzrcvokdzogaulsjgagcxqnlpakqsfrarznpdhchkaopvfuiwjjgx
zpduqwmxczjwrchgxwtfadfoeuypkxtfblbhcvpgzuksbeploiqrrkdjtwupouhcoawyavekomhsfoqhpiqpvsxjqsxsvphwstvqvedavqiplhmdmyojegbvhqkkespkqbsfvzftuwzidadqpbskhucvzoefxynufcoshkjbmzoirnnbeirzvalqvuqtgjrysdksevhqtecduscakzmjorjbaxsdesqqltlkuucqtsilzubrvbveioyardgoryv
ultdanqyblzhklqleghmoahvmfylpfmelsnehzicopzsnwhzmgfsvtimmbqungivpwrdzjlbglidtnkdgzaqfzlohgpuzmjcxudbhmipfmqbcmffqtktwffuhsqewyunyqtjgquyahoprfqipbamrxjuzemyzlnmqeyfgvbfhinybirwulbwbjrcojurbitdnsmivwyapgxjbnlaqrismaepmmlvqhyrrcymaqsvchsgjjdokrcforymoynmwqz
cwbosooclvfffnvhrdkqmfextqtaaxgnqlrxlunaowiksuxhirsrtqreonhnclghnaoxxossrhzaixdbswzixmtbbsmwsttvepcvvejiwhpdhmgurkmhpnkjjeesirvfyufxcpwythjhwiafeimzsifxdicjyofhiwiljpcjfhgidikvdohakyiirczkikfpwgctweqbdzdmetqkkbiylmznulqlqaeeqdtngpgtvlaupgsiraskavxrrrdhmhh
uwflaqlrklhpqizwvfkhhxvfxrqyacopmssqlqtfnmcvcdfsbfinnqwagjgdsipjoxndgekghsglgxkogcjnrdykhgvqueaghjjjrovasugeywqcohktxdxvoryqtqoznvbvaufvsbwqugspfmnqumofiecgkgggznsktjppqztiytwtixjeykzvldmmyylimcbvepumooscrkwbambxaxgckyqdycejoljmvnvcbucdxhglshseshzmyhdcchy
djxvygqasegdqnarevifrpbrzhrikbeqersxkeyjurwnhtthgirkivmvtkdnkeenqjoxhlocqdobdsawbnijvjglsyqjpesrqhnnnfpwxdhskwuxqtahnenzyihwjgjpkapakidotxweryfbazlpcxjnpjwfaqpwymwvgirjaxmjdpwljwlvfudojowjsxutgzyequzpzylnlkjpabyxhjagxhasrurynbtqpzsxamxazxmdkfmklwoijerfwdt
dioehsxvmgyjfrxaarqwttvbhzqeollhwhimwrwhhnikkpcnfdcnmolcwsixgxgmljfjzfsdcsvmnmdmnmelkbmcdzatgsqkwtfuftbiapdjtlotjncmxilcjdvxyforrjbbyhvsqsmqezvdrnqxnzqpdrerbizohkvuivquagwxuajbvrphjknbfoojykvawxrkkmvczmfvobzpdnhdufbvnlkomxbrmabywmbulyutdzovyyfldbprqnktwfd
jiguwtqrcxxdnjepmbriuzfmpyzsdwbmgqilugruhmqmohsqocdxfnngyhbltsfujgrnijcjctyvhtcudutaqzndpuetdkjlkfgvslpylrhcrcdagvbizdxnvwbxxyjxbwznyvwgadqmjiyodqdxgthkkyvwnvvzdlqmzsitglpjaqndukjrezexqgbcieohwukzkehzddyukkcklpyzgzdtynejcgusildwtrmaghsiblzejirgegnubehyecn
rmqdfuubgxbhkspirjcavqgswsdsywxbzccxoztkfjraznodfzmauirfodgcducpkfymrkysnylswyfpqhbfwcyemagdpnkuognflmwpqwgvwqrlwrkookxigwuhwwxqeknypvhliexryrccdchqfqwrwyglxbngyakngeewufitggtyogkxrfnkvtkobsgbgfckwavfxrjpshhnenqdmowjvtdmzxdncpcxjhmmvsukvvfbgavgpkhzwudclkd
pxykzitkwodhkqtxzxkqqqercpvfqmeumvhhqjnaxgarrnffqynycfwqizqclzxqiaubpzujmqfkqqrmlvqcrzcebuygsvucbrvyzmtrnxludhlekcwrgzenwythbjodycgjcgswhxtvputyyzfsfxvlwobobxnfudmubbggcmlxgsxbxwrpscgiockscxpqndodxketpqcravcgizjlzomhkdqrugbzyamukepcjespuxyyqkktnaemfsgjaqo
zamtsncdsdozwbaiybkqlxqaiuxrhzxiighfglgrtaakdgnahwzrwrfjlbetaeddnpwnmwowjkpdpjdmyhnrhlszgunotskusdnnkmifrlzzjdqlhhletkskvipsvrweeteqaaijlhbwhdyelikprzevkxlrsgaikhzqdfmzvmimnaehglojyvsbdhnnyfdbrqxscgxmkvzpsarzfmjxjtzmexoecywjjfjshmckrlqbqfnshsdprumxorsxvvr
wnpxtyveagdywgkafkhjdiwiwszxouwkwisuylfgbqymsyzqyzvheosxjrpdlywbfsbbixffojkvxktndwkzbwyoywedmemuideumiwckggidyvmrthnpzcebxjxfalhatajqclihzwarvxdkkxmgojnzyrszstaxlbopxnfbhhqjetmipcpiylesdqewrrvyboqblnkcfbxuxzhfuratapmyvsfaceixlzxpydwpebtzpxfniujwelogzhdgcb
gpntrsaileyuocchttrrgnlglpvwuelkbgzfglymxfkfntsqvldqbfpksmhdhwdgntytgbklnfacvjocmjtotbsuxhvnubmwvucbhzywunlvuuifjlaztyuysbzpkiwynlshvkzcxgesqbwixwjtjrbpihgwskyuulecdvkvvgnpdcxxvefxlqmnswrjgyvrqzlhkybfesmzbeldziczgsocngmmimjgxagjrdlbexboygdhyixnqyoxtzbwsfi
jgponelocagsqilrczgoblewzieothrsxmmanfudlpexvagwblrttzrvclvxoxlqkodqsgljthpudnynxzuthhbirculzslpznjbgvkwgxpgxpeghkkgvavugjwzdmxeqwsipjesgmjjxoyjzaavkkwnfhqbnbprmizeyolyxeyqsztrxnolsoijegmxntfxtcvfgkubzqortzpekdwopmsezkbjfumeanrnoufdiyaoqpnjillnusppaghbeeu
firkdxdoitnbkcuyzklwwqwyezncibuckdcnihnfgshznampynplssckmhvrlzqtcfwhohecjwibongpuehxkbwelnxqkxyqsikylbesmhmnsjyeegjkdvwhlyuggrxffwazgbhoagjqrbvofwgvvxhvekvuguylgzwfogxesbthfakfvfbvcqyyabrmnpdswzowpvufienyurccjueogkphmmznrlshnmgxlogkbbtajbeftqtrgearargqjpa
uturffydzaggeezzehjbqxgsjomemphnpjarxguijflzvfghpekjlmmgdehcuwcylnotlzsaakcjavsqgyslcwvuwsbczrfulnhiujqjvtynbtohmiftauyhqsghgptjibnfchfepjnwmzprvdisrmxefsxxouirfchqtsxtapqzqsfchoelvzzqgeogjyhcqcpolgduvlqrtlefnjukhwkqodyohtbifdbpvlgmywqtclljbalgmfkdsstbrra
fpvsqifeblhosznczjnymjkkdegnugahelgcphtmaaepelcwsmoidbaqfedmgurhjlkhnrmhqzaymemmblskcvmewggjcpmivalsbmtvwkkqadnzzwmohwhpxuspqipjrnnqjubedeeyyacpowcpvxatfkcmksgibvdelwbbizrqcnukysyzwapxtnxeocaucsofehlizndbkmlugdjmckhavnjzjubauqnigcmfwdwaofakrdsogvpcoilynhi
lwzlwiqwuhvujakqsqodocushuhanuldzbidygndloffgndjujqgojrdadcmsjriqeiambnirxtnvggeebhixqfllaqcxsiulrkdduawlxetxlpwtgdngvsfidkedhsgmtavnrjmxdhvfbesyhnnhofeftkovsavskltojjxfuutsavyjjsiusbmdbqzandyiwzzfkuvbrjfcrdmxgbcejzpxsjiwpsjckfmsylncuehurvlfbqzybguqbnvrus
trwdpdswpwrqpjiossssfxdnwjoqtkubutrmggaezltensaprgiaesf
mm