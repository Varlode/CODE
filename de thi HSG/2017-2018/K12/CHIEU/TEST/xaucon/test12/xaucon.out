558 1371 1749 1788 2117 3099 3109 3197 3426 4431 4549 5921 6082 6562 7626 8634 8704 9142 9234 9514 